src/main/js/wiki/containers/Settings.tsx

```diff
@@ -53,14 +54,27 @@ const Settings: FC<Props> = (props) => {
   const rootDirValid = isValidRelativePath(rootDir);
   const landingPageValid = isValidRelativePath(landingPage);
   const allValid = rootDirValid && landingPageValid;
+  let historyLink = "#";
+
+  function findDirectoryPath(props) {
```

Comment:
This methods already exists in the history component. Place it at a single location and export it to follow the D.R.Y. principle

Reply:
- [ ] done.

Is an export from within `Settings.tsx` okay?